test/acceptance/user.controller.acceptance.ts

```diff
@@ -141,11 +141,17 @@ describe('UserController', () => {
   describe('authentication functions', () => {
     let plainPassword: string;
     let jwt_auth_service: JWTAuthenticationService;
+    let bcrypt_hasher: BcryptHasher;
 
     before('create new user', async () => {
+      bcrypt_hasher = new BcryptHasher(4);
```

Comment:
Since this is acceptance-level test, behavior should be configured mostly at app level via bindings. 

I think you should change `BcryptHasherBindings.ROUNDS` in one of the early hooks to ensure that all tests are using less rounds to hash password when creating new users.